src/rules/no-absolute-path.js

```diff
@@ -10,13 +11,25 @@ module.exports = {
       description: 'Forbid import of modules using absolute paths.',
       url: docsUrl('no-absolute-path'),
     },
+    fixable: 'code',
     schema: [ makeOptionsSchema() ],
   },
 
   create(context) {
     function reportIfAbsolute(source) {
       if (isAbsolute(source.value)) {
-        context.report(source, 'Do not import modules using an absolute path');
+        context.report({
+          node: source,
+          message: 'Do not import modules using an absolute path',
+          fix: fixer => {
+            const resolvedContext = context.getPhysicalFilename ? context.getPhysicalFilename() : context.getFilename();
+            let relativePath = path.relative(path.dirname(resolvedContext), source.value);
+            if (!relativePath.startsWith('.')){
+              relativePath = '.' + path.sep + relativePath;
+            }
```

Comment:
```suggestion
            if (!relativePath.startsWith('.')) {
              relativePath = path.join('.', relativePath);
            }
```

Reply:
`path.join('.', relativePath)` drops the leading `.`, this is why I chose to do explicit string concatenation.

Reply:
ah ok, fair enough